examples/server/server.cpp

```diff
@@ -919,13 +919,34 @@ int main(int argc, char ** argv) {
         } else if (params.response_format == vjson_format) {
             /* try to match openai/whisper's Python format */
             std::string results = output_str(ctx, params, pcmf32s);
+            
```

Comment:
Nit: Remove empty spaces  (you can't see them here but they show up in local diffs as red bars, and it is just nice to not have the extra "noise").